openapi_specs/openapi-modified.yaml

```diff
@@ -2435,6 +3048,16 @@ components:
         - messages
         - agent_id
       title: AgentsCompletionRequest
+    ContentChunk:
+      oneOf:
```

Comment:
anyOf here gives better results but not fully functional.

Reply:
Let's leave it as it is for now. If there will be problem then will address it in next release.